### DISCUSSION TITLE
Transferring arrow files between projects

### DISCUSSION THREAD

**Discussion by boconnell89** [GreenleafLab/ArchR#845] [Questions / Documentation]
Is there a way to create an `ArchRProject` with `ArrowFiles` created for another project (with the same reference genome, etc)? I'm trying to avoid having to process some very large files from different sources, that I wish to load into an `ArchRProject` to compare. Each of these has takes about a day to create an `ArrowFile`, so I'd like to avoid that if possible.

**Comment by rcorces** [comment]
Arrow files can be copied and used in multiple projects. Problems will arise if multiple projects share the same copy of an arrow file though. So we typically create the base Arrow file and then create an ArchRProject using `copyArrows = TRUE` to prevent the project from editing the original arrow file copy.
